providers/implementations/signature/ml_dsa_sig.c

```diff
@@ -54,13 +58,20 @@ typedef struct {
     uint8_t aid_buf[OSSL_MAX_ALGORITHM_ID_SIZE];
     size_t  aid_len;
     int mu;     /* Flag indicating we should begin from \mu, not the message */
+
+    int operation;
+    EVP_MD_CTX *md_ctx; /* Ctx for msg_init/update/final interface */
+    unsigned char *sig; /* Signature, for verification */
+    size_t siglen;
```

Comment:
be consistent with rest of file sig_len.

Reply:
most functions take "siglen", and this is the same name that exist in PROV_RSA_CTX and PROV_ECDSA_CTX so I tried to stay consistent with the other implementations.